src/game/adapter.rs

```diff
@@ -12,6 +12,9 @@ use tokio::sync::broadcast;
 pub enum GameAdapterError {
     PlayerLimitExceeded(usize),
     InvalidGameState(State),
+    WrongPlayerRequest(String),
+    WrongMoveRequest(usize),
+    GameEnded(),
```

Comment:
isn't this a special case of `InvalidGameState(Ended)`?

Reply:
Yeah it is, I was talking to Johnathan and he told me we should print a different message when sb tries to make a move after the game has ended, I changed it accordingly.

If u want i can revert it back the way it was, without that special case